src/uu/sort/src/sort.rs

```diff
@@ -85,6 +91,14 @@ static THOUSANDS_SEP: char = ',';
 static NEGATIVE: char = '-';
 static POSITIVE: char = '+';
 
+#[cfg(any(target_os = "windows",))]
+static DEFAULT_TMPDIR: &str = r"%USERPROFILE%\AppData\Local\Temp";
+
+#[cfg(not(any(target_os = "windows",)))]
+static DEFAULT_TMPDIR: &str = r"/tmp";
```

Comment:
we cannot use https://doc.rust-lang.org/std/env/fn.temp_dir.html here?

Reply:
We do so now.  These const defaults are now the 3rd option final option after 1. user specified, and 2.  std::env::temp_dir.  And I think you're right, these are superfluous and we can remove now since I believe std::env::temp_dir guarantees not to return an empty string.  Thanks.